src/ai.ts

```diff
@@ -20,6 +21,15 @@ const LLM_MODELS = [
     name: "gpt-4o-mini",
     createAi: createOpenAI,
   },
+  // Google
+  {
+    name: "gemini-1.5-flash",
+    createAi: createGoogleGenerativeAI,
+  },
+  {
+    name: "gemini-1.5-flash-latest",
+    createAi: createGoogleGenerativeAI,
+  },
```

Comment:
Consider adding version validation or error handling for the Gemini models. Since there are two versions (regular and 'latest'), it would be helpful to:
1. Add comments explaining the difference between these versions
2. Implement version checking to ensure compatibility
3. Consider adding fallback logic if a specific version is not available